site/en/workflows.md

```diff
@@ -1376,29 +1569,10 @@ If your computer is not that powerful you may try this workflow. It works fine f
     </tr>
```

Comment:
Step 3 skimage-binarize is still in here (and also in the command list below). This is _slow_ and not strictly necessary. Just a reminder in case you did forget (cf. [my comment last time](https://github.com/OCR-D/ocrd-website/pull/189#discussion_r536159259)).